components/inputs/README.md

```diff
@@ -9,6 +9,87 @@ There are various input components available:
 - [Checkboxes](#checkboxes)
 - [Radio Buttons](#radio-buttons)
 
+## Labelling Inputs
+
+All inputs *must* have a label. Web component-based inputs like `<d2l-input-checkbox>`, `<d2l-input-search>` and `<d2l-input-text>` come with built-in labels. For the rest, labelling is accomplished visually using a `<label>` element or with a hidden label via the `aria-label` attribute.
+
+Groups of inputs (like checkboxes or radios) should be wrapped in a `<fieldset>` which can have label styles applied to it.
+
+### Visible labels using the `<label>` element
+
+![example screenshot of input label](./screenshots/label.gif?raw=true)
```

Comment:
It looks like the screenshots in this PR are actually png so these are broken.

Reply:
Good catch -- fixed!